src/components/AddItem.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export default function AddItem({ listPath }) {
+	const [itemValue, setItemValue] = useState({
+		itemName: '',
+		daysUntilNextPurchase: 0,
+	});
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			alert('Item added!');
+			await addItem(listPath, itemValue);
+		} catch (err) {
+			alert('Error adding item to database');
+		}
+	};
```

Comment:
does this need a return statement?

Reply:
I don't think so because it's not returning anything, just creating a side effect (the alert). But now that you had me looking at this code again, I realized the statements in the `try` block need to be swapped - I just created a new commit for that.

Reply:
Based on MDN ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return ) I agree we do not specifically need return because there is no return statement.